src/components/ThumbnailsPanel/ThumbnailsPanel.js

```diff
@@ -28,6 +32,9 @@ class ThumbnailsPanel extends React.PureComponent {
       canLoad: true,
       height: 0,
       width: 0,
+      draggingOverPageIndex: null,
+      isDraggingOverTopHalf: false,
+      afterMovePageNumber: null,
```

Comment:
why is this on state? Does this variable related to how the thumbnail panel looks like?

Reply:
I moved the "afterMovingPageNumber" out of the state. It not used for appearance so it shouldn't be there